pkgs/desktops/plasma-5/kde-cli-tools.nix

```diff
@@ -11,4 +11,12 @@ mkDerivation {
     kcmutils kconfig kdesu ki18n kiconthemes kinit kio kwindowsystem qtsvg
     qtx11extras kactivities plasma-workspace
   ];
+  postInstall = ''
+    cat > $out/bin/kdesu <<EOF
+    #! /bin/sh -e
+    export PATH="\''${PATH}:${kinit}/bin"
+    exec -a "\$0" $out/libexec/kf5/kdesu "\$@"
+    EOF
+    chmod +x $out/bin/kdesu
+  '';
```

Comment:
What is the need for this?

Reply:
- have `kdesu` in `$PATH`
- avoid an error about `kinit` not found when running outside of plasma

Reply:
This is a test running `kdesu` in a VM under fluxbox. Running the original `kdesu` executable from `libexec` fails. But running the script this PR install under `bin` succeeds.

```
[guest@nixos:~]$ /nix/store/kxrmiw71jk2hs5qb65q18dyddjcxiwki-kde-cli-tools-5.17.5/libexec/kf5/kdesu xterm
kf5.kdbusaddons: Can not find 'kdeinit5' executable at  "/run/wrappers/bin:/home/guest/.nix-profile/bin:/etc/profiles/per-user/guest/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin" "/nix/store/kxrmiw71jk2hs5qb65q18dyddjcxiwki-kde-cli-tools-5.17.5/libexec/kf5, /nix/store/ymsw8j9876ivxirmyllflxgf0gpdx1d7-qtbase-5.12.7/bin"
"KLauncher could not be reached via D-Bus. Error when calling kdeinit_exec_wait:\nThe name org.kde.klauncher5 was not provided by any .service files\n"

[guest@nixos:~]$ /nix/store/kxrmiw71jk2hs5qb65q18dyddjcxiwki-kde-cli-tools-5.17.5/bin/kdesu xterm
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)


```

Reply:
It seems that `kdeinit5` is a run time dependency that should be found in `$PATH`.